full-v3.yml

```diff
@@ -0,0 +1,2299 @@
+      operationId: getContactsForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/ContactsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/mystudents:
+    get:
+      tags: ["Users"]
+      description: Returns the students for a user
+      operationId: getStudentsForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/StudentsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/myteachers:
+    get:
+      tags: ["Users"]
+      description: Returns the teachers for a user
+      operationId: getTeachersForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/TeachersResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/resources:
+    get:
+      tags: ["Users"]
+      description: Returns the resources for a user
+      operationId: getResourcesForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/ResourcesResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/schools:
+    get:
+      tags: ["Users"]
+      description: Returns the schools for a user
+      operationId: getSchoolsForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
```

Comment:
Is this necessary? Will a user have that many schools?

Reply:
I guess same question for /users/{id}/sections. If we do this for all endpoints that can return more than one result, then feel free to ignore.

Reply:
A district admin could have a lot of schools, and some districts have custom teachers in every section. Regarding question 2, we had that for, e.g., /teachers/{id}/schools and /teachers/{id}/sections before. Planning to keep that format.